modules/ROOT/pages/upgrade-pre-requisites.adoc

```diff
@@ -0,0 +1,19 @@
+= Upgrade Pre-requisites 
+
+For all OpenShift upgrades from versions 2.6.52 and earlier to versions 2.7.0 and later, you must manually increase the `rtfd` container resources to the desired values.
+
+To update `rtfd` container resources: 
+
+. In the Red Hat OpenShift console, navigate to *Operators* > *Installed Operators*.
+. Select the *Runtime Fabric* operator and open the Runtime Fabric instance `.yaml` file.
++
+image::runtime-fabric-yaml.png[Runtime Fabric instance .yaml file.]
++
+. In the `.yaml` file, go to  `spec:` > `agent:` > `rtfd:`, and update the resource allocations to recommended values.
++
+image::rtfd-change-values.png[Update rtfd resource allocations to recommended values.]
++
+. Click *Save*.
+
+[NOTE]
+For Helm-based upgrades, if the helm-based installations had previously overridden `rtfd` container resources in the `values.yaml` file, then you must manually increase the allocations as per scalability guidelines, as Helm-based upgrades respect pre-release manual overrides.
```

Comment:
```suggestion
For Helm-based upgrades, if the Helm-based installations had previously overridden `rtfd` container resources in the `values.yaml` file, then you must manually increase the allocations as per scalability guidelines, as Helm-based upgrades respect pre-release manual overrides.
```

Reply:
This one is still a little long, and I'm not sure what you mean by "respect pre-release manual overrides". Should this be changed to:

"retain manual overrides made in the previously installed version"?